### PR TITLE
feat(ingest): Add support for user to delete the ingested items

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: superdesk/io/ingest.py [hunk 1]
```diff
@@ -62,3 +66,17 @@ def set_ingest_provider_sequence(self, item, provider):
             max_seq_number=app.config['MAX_VALUE_OF_INGEST_SEQUENCE']
         )
         item['ingest_provider_sequence'] = str(sequence_number)
+
+    def on_deleted(self, docs):
```
Comment (mdhaman) [inline superdesk/io/ingest.py]: a behave test would be good.

File: superdesk/io/ingest.py [hunk 2]
```diff
@@ -62,3 +66,17 @@ def set_ingest_provider_sequence(self, item, provider):
...
+                    for doc in docs
+                    for rend in doc.get('renditions', {}).values()
+                    if not doc.get('archived') and rend.get('media')]
+
+        for file_id in file_ids:
+            superdesk.app.media.delete(file_id)
+
+        user = get_user(required=True)
+        if docs:
+            push_notification('item:spike', item=str(docs[0].get(config.ID_FIELD)), user=str(user))
```
Comment (mdhaman) [inline superdesk/io/ingest.py]: did you meant `item:delete`

**Comment by petrjasek** [comment]
:+1: behave tests would be great

File: superdesk/io/ingest.py (hunk 1 above)
Comment (petrjasek) [inline superdesk/io/ingest.py]: didn't see the comment when writing mine ;)

**Comment by akintolga** [comment]
What happens when an ingested package is deleted?

File: superdesk/io/ingest.py (hunk 1 above)
Comment (ioanpocol) [inline superdesk/io/ingest.py]: done

**Comment by ioanpocol** [comment]
fixed for packages and added tests

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**